settings/src/components/webauthn/webauthn.scss

```diff
@@ -1,3 +1,20 @@
+.wporg-2fa__webauthn-keys-list {
+	li {
```

Comment:
The title spacing and this list are bigger than in the designs:


| This PR | Design |
| --- | --- |
| <img width="753" alt="Screenshot 2023-08-11 at 3 25 43 PM" src="https://github.com/WordPress/wporg-two-factor/assets/1657336/c5bbb543-9da9-41a8-83d2-3653a72beec0"> | <img width="408" alt="Screenshot 2023-08-11 at 3 28 24 PM" src="https://github.com/WordPress/wporg-two-factor/assets/1657336/8fcfbdc2-d229-40cd-8b7e-6cdb0e6bb5e6"> |

Reply:
Removed the space